xml/hwbp_registers.xml

```diff
@@ -33,6 +33,10 @@
     <register name="Trigger Data 1" short="tdata1" address="0x7a1">
         This register is optional if no triggers are implemented.
 
+        Writing 0 to this register must result in a trigger that is disabled.
+        If this trigger supports multiple types, then this should be done by
+        changing \FcsrTdataOneType to 8.
```

Comment:
This should make clear that the change should be made by hardware as part of the WARL process, not by the debugger.

Reply:
Done.